File: practise/nqtquestion1.py
i = int(input())

def cal_price(i):
    if i<=2:
        price = 100*i
    elif i<=5:
        k = i - 2
        price = 2*100 + k*50
    else:
        k = i-5
        price = 2*100 + 3*50 + k*20
    print(price)

cal_price(i)

File: practise/nqtquestion1.cpp
#include <iostream>
using namespace std;

int cal_price(int i) {
    int price;
    if (i<=2){
        price = i*100;
    }
    else if (i>=2 && i<=5){
        price = 2*100 + (i-2)*50;
    }
    else {
        price = 2*100 + 3*50 + (i-5)*20;
    }
    cout<< price;
}
int main(){
    int i;
    cin >> i;
    cal_price(i);
}